packages/core/src/components/TextField.tsx

```diff
@@ -68,7 +68,7 @@ interface State {
   value?: string;
 }
 
-class TextField extends React.Component<Props, State> {
+class StyledTextField extends React.Component<Props, State> {
```

Comment:
Can we keep the tag for this as `TextField` under the hood, so that existing TextField components keep working in Draftbit when we upgrade the version of Jigsaw?

```suggestion
class TextField extends React.Component<Props, State> {
```

Reply:
Updated